isaric/parsers/brazil-followup.toml

```diff
@@ -211,6 +211,15 @@
   description = "Admission to an Intensive Care Unit (ICU / CTI)"
   ref = "Y/N"
 
+[subject.vaccinated_covid19]
+  field = "flw_cov_vacc"
+  values = { 1 = true, 0 = false }
+
+[subject.vaccinated_covid19_dates]
+  combinedType = "list"
+  excludeWhen = "none"
+  fields = [{ field = "flw_vacc_last_dose" }]
```

Comment:
add context or similar to indicate this is date of most recent dose, rather than dates of all vaccinations received?